dataland-frontend/.env

```diff
@@ -1,4 +1,4 @@
 #Dataland settings to source into frontend code
 VITE_DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES=${DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES}
 VITE_DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES=${DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_MEGABYTES}
-VITE_MAX_NUMBER_OF_DAYS_SELECTABLE_FOR_API_KEY_VALIDITY=${MAX_NUMBER_OF_DAYS_SELECTABLE_FOR_API_KEY_VALIDITY}
+VITE_MAX_NUMBER_OF_DAYS_SELECTABLE_FOR_API_KEY_VALIDITY=${MAX_NUMBER_OF_DAYS_SELECTABLE_FOR_API_KEY_VALIDITY}
```

Comment:
there should not be any change to this file in the final version

Reply:
Why is this flagged as changed anyway? Seems rather identical

Reply:
i added and removed a line in some previous commits, but have on idea actually